Mathlib/RingTheory/Ideal/LocalRing.lean

```diff
@@ -199,71 +196,40 @@ end LocalRing
 
 end CommRing
 
-/-- A local ring homomorphism is a homomorphism `f` between local rings such that `a` in the domain
-  is a unit if `f a` is a unit for any `a`. See `LocalRing.local_hom_TFAE` for other equivalent
-  definitions. -/
-class IsLocalRingHom [Semiring R] [Semiring S] (f : R →+* S) : Prop where
-  /-- A local ring homomorphism `f : R ⟶ S` will send nonunits of `R` to nonunits of `S`. -/
-  map_nonunit : ∀ a, IsUnit (f a) → IsUnit a
-#align is_local_ring_hom IsLocalRingHom
-
 section
 
-variable [Semiring R] [Semiring S] [Semiring T]
+/-- If `f : R →+* S` is a local ring hom, then `R` is a local ring if `S` is. -/
+theorem map_local_ring [CommSemiring R] [CommSemiring S] [LocalRing S]
```

Comment:
Is this the correct name? Shouldn't it be `map_localRing`?
Or even `localRing_of_isLocalRingHom`...

Reply:
Yes, I guess it should be `map_localRing`. Changed. If they both fit the naming scheme, I tend to prefer shorter names, personally - I'm not sure if there's opinions about this.